### DISCUSSION TITLE
Is there a commandline option to select graphics card?

### DISCUSSION THREAD

**Discussion by Drael64** [ggml-org/llama.cpp#3367] [Q&A]
Sorry if this sort of question isn't allowed here! 

I'm getting a decent gaming mini-pc with a dgpu and an igpu. Igpu might be slow but offers the possibility of much more ram for things like quantization. Can you select a graphics card between two options for things like the quantize module?

**Comment by staviq** [comment]
When you compile ( or download ) llama.cpp with gpu support, you compile ( or download ) it for a particular gpu family, so unless you have the same kind of igpu and dgpu, it will already only see one of them, depending on gpu target it was compiled for.

Though typically igpus aren't capable of compute the same way dgpus are, it really depends on the particular hardware model.

**Comment by Drael64** [comment]
Hmm, so the igpu is a 680M, and the dgpu is a 6600M, they are both from AMD.
